src/test/java/redis/clients/jedis/doc/example.json

```diff
@@ -0,0 +1,4 @@
+{
+    "id": "auto",
+    "file": "*Example.java"
+}
```

Comment:
What is the purpose of this file?

Reply:
It was initially used by the tool/script that builds the documentation site to discover examples. The latest version just looks for all files in this folder and checks for the `//EXAMPLE: $id` comment in the first line. So, I think that this can be removed in the upstream repo (your repo).